x-pack/plugins/uptime/server/lib/adapters/pings/elasticsearch_pings_adapter.ts

```diff
@@ -181,36 +178,24 @@ export class ElasticsearchPingsAdapter implements UMPingsAdapter {
               buckets: 25,
             },
             aggs: {
-              by_id: {
-                terms: {
-                  field: 'monitor.id',
-                  size: 200,
+              down: {
```

Comment:
So much simpler and more efficient :)

Reply:
Felt real good to delete some of this code